pkg/url/url_test.go

```diff
@@ -1645,17 +1722,23 @@ func TestPush(t *testing.T) {
 			existingEnvInfoURLs: []envinfo.EnvInfoURL{
 				{
 					Name: "example",
-					Port: 8080,
 					Host: "com",
 					Kind: envinfo.INGRESS,
 				},
 			},
-			endpintMap: map[int32]versionsCommon.Endpoint{
-				8080: versionsCommon.Endpoint{
-					Name:       "example",
-					TargetPort: 8080,
-					Secure:     false,
-					Path:       "/testpath",
+			containerComponents: []versionsCommon.DevfileComponent{
+				{
+					Container: &versionsCommon.Container{
+						Name: "container1",
+						Endpoints: []versionsCommon.Endpoint{
+							versionsCommon.Endpoint{
```

Comment:
we don't need to type `versionsCommon.Endpoint` here and in the above cases

Reply:
updated